reporters/kamon-opentelemetry/src/main/resources/reference.conf

```diff
@@ -14,25 +14,64 @@ kamon.otel.trace {
   # https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md
   endpoint = ${kamon.otel.trace.protocol}"://"${kamon.otel.trace.host}":"${kamon.otel.trace.port}
   endpoint = ${?OTEL_EXPORTER_OTLP_ENDPOINT}
-  endpoint = ${?OTEL_EXPORTER_OTLP_TRACES_ENDPOINT}
 
-  # Enable or disable including tags from kamon.environment.tags as resource labels in the exported trace
-  # Any keys containing a hyphen (-) will be converted to a dot (.) as that is the standard.
-  # e.g. service-version becomes service.version
-  # reference: https://github.com/kamon-io/Kamon/blob/master/core/kamon-core/src/main/resources/reference.conf#L23
+  # Supports empty string or gzip
+  compression = ""
+  compression = ${?OTEL_EXPORTER_OTLP_COMPRESSION}
+
+  # Supports comma-separated pairs (i.e "api-key=supersecret,data-type=application-traces")
+  headers = ""
+  headers = ${?OTEL_EXPORTER_OTLP_HEADERS}
+
+  timeout = 10s
+  timeout = ${?OTEL_EXPORTER_OTLP_TIMEOUT}
+
+  # Supports grpc and http/protobuf
+  otelProtocol = "grpc"
+  otelProtocol = ${?OTEL_EXPORTER_OTLP_PROTOCOL}
```

Comment:
There is another plain `protocol` setting in the config file, can you remove it? I think it is not being used anywhere.

Reply:
Also, please change `otelProtocol` and `fullEndpoint` to kebab-case.